config/management/src/json_rpc.rs

```diff
@@ -48,6 +49,32 @@ impl JsonRpcClientWrapper {
         .ok_or_else(|| Error::JsonRpcReadError("validator-config", "not present".to_string()))
     }
 
+    pub fn validator_info(&self, account: AccountAddress) -> Result<ValidatorInfo, Error> {
+        let validator_set_account = account_config::validator_set_address();
+        let validator_set = self
+            .account_state(validator_set_account)?
+            .get_validator_set();
+
+        match validator_set {
+            Ok(validator_set) => match validator_set {
+                Some(validator_set) => validator_set
+                    .payload()
+                    .iter()
+                    .find(|validator_info| validator_info.account_address() == &account)
+                    .cloned()
+                    .ok_or(Error::JsonRpcReadError(
+                        "ValidatorInfo",
+                        format!("{} is not found", account),
+                    )),
+                None => Err(Error::JsonRpcReadError(
+                    "ValidatorSet is not set",
+                    String::new(),
+                )),
+            },
+            Err(e) => Err(Error::JsonRpcReadError("ValidatorSet", e.to_string())),
+        }
```

Comment:
Nit: I mentioned this on Josh's too, I think you guys implemented the same function.  You can do a multi level match to make it a little cleaner

```
match validator_set {
  Ok(Some(validator_set)) => { ... }
  Ok(None) => { ... }
  Err(e) => { ... }
}
```